src/haxeLanguageServer/Context.hx

```diff
@@ -154,7 +154,7 @@ class Context {
         workspacePath = params.rootUri.toFsPath();
         capabilities = params.capabilities;
         var options = (params.initializationOptions : InitOptions);
-        if (options == null) {
+        if (options == null || Reflect.fields(options).length == 0 ) {
```

Comment:
Looks like neovim was passing in an empty object for options when the language server starts initially.  The null check doesn't look like it's enough, we need to populate defaults if the option is empty.